util/stmtsummary/statement_summary.go

```diff
@@ -627,8 +641,10 @@ func newStmtSummaryByDigestElement(sei *StmtExecInfo, beginTime int64, intervalS
 		firstSeen:    sei.StartTime,
 		lastSeen:     sei.StartTime,
 		backoffTypes: make(map[fmt.Stringer]int),
+		authUsers:    make(map[string]struct{}),
```

Comment:
If `authUsers` is added, `sampleUser` can be removed.
When output `sampleUser`, you can get one from `authUsers`.